Content.Client/GameObjects/Components/Mobs/ClientStatusEffectsComponent.cs

```diff
@@ -129,7 +129,10 @@ public void FrameUpdate(float frameTime)
         {
             foreach (var (effect, cooldownGraphic) in _cooldown)
             {
-                var status = _status[effect];
+                if(!_status.TryGetValue(effect, out var status))
```

Comment:
Shouldn't this be removed from the list if this is the case? Why's it even in there still?

Reply:
¯\_(ツ)_/¯

Reply:
You know I'm not gonna merge a PR with that kind of reasoning. This could easily be a symptom of a great bug and just muting the error like this may not be the correct solution.

Reply:
I'm shrugging because I literally don't know how to fix it. Feel free to close the PR!